memote/support/sbo.py

```diff
@@ -67,4 +67,4 @@ def check_component_for_specific_sbo_term(items, term):
     return [elem for elem in items if
             elem.annotation is None or
             'sbo' not in elem.annotation or
-            elem.annotation['sbo'] not in term]
+            term not in elem.annotation['sbo']]
```

Comment:
Unfortunately, I don't think this fixes the issue entirely: 
- `test_transport_reaction_specific_sbo_presence` checks for a list of possible SBO terms related to transport reactions i.e. `term` can be a list of strings.
- The tests don't seem to cover that specific case

Would you be OK with taking that on? I realize that this is more than you bargained for.

Reply:
Really the whole SBO is a temporary stand-in for a solution handling the ontology as suggested in #469